client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -100,6 +103,10 @@ const ProductLightbox: React.FC< Props > = ( {
 
 	const isMultiSiteIncompatible = isMultisite && ! getIsMultisiteCompatible( product );
 
+	const includedProductSlugs = getSlugsOfProductsInludedInBundle( product );
+
+	const isLargeScreen = useBreakpoint( '>782px' );
```

Comment:
is there a reason why we don't use CSS media query?

Reply:
The reason for that is the order of the items on desktop and mobile. In desktop view, the breakdown is inside the sidebar of lightbox, while as in mobile view it's in details section, right under the description.

Reply:
Ahh I see what you mean. I was thinking of having two `<PriceBreakdown />` exist both in the main container and side panel and let CSS dictates which one to show base on the current viewpoint. I'm not sure if this approach is more performant compared to react render. But I agree with this approach as well.

Reply:
Yeah, I agree with both approaches. It's just that I preferred conditional rendering here.